extension/css/cryptup.css

```diff
@@ -1309,35 +1298,15 @@ td {
   max-width: 300px;
   float: right;
   text-align: right;
-  opacity: 0.5;
+  opacity: 1;
   margin-left: 1em;
 }
 
-#pgp_signature.rtl {
```

Comment:
do we still need `rtl` class for signature?

Reply:
I suppose no. Can treat it the same.